script.js

```diff
@@ -0,0 +1,96 @@
+// send login details
+function sendLoginDetails() {
+    var userName = document.getElementById("userName");
+    var password = document.getElementById("password");
+
+    var form = new FormData();
+    form.append("userName", userName.value);
+    form.append("password", password.value);
+
+    var request = new XMLHttpRequest();
+    request.onreadystatechange = function() {
```

Comment:
Use arrow function ()=>{} for this anonymous function

Reply:
OK